Gds/src/fprime_gds/common/adapters/ip.py

```diff
@@ -46,7 +46,9 @@ class IpAdapter(fprime_gds.common.adapters.base.BaseAdapter):
     KEEPALIVE_INTERVAL = (
         0.500  # Interval to send a KEEPALIVE packet. None will turn off KEEPALIVE.
     )
-    KEEPALIVE_DATA = b"sitting well"  # Data to send out as part of the KEEPALIVE packet. Should not be null nor empty.
+    KEEPALIVE_DATA = (
```

Comment:
Why? This the format tool?

Reply:
Yes the format tool regards the comment as an important part of that line so it formats to keep it 'on' the same line